### PR TITLE
fix(axios-extension): message bas deploy

### PR DESCRIPTION
**Pull request by mikicvi-SAP**
BUG - Wrong link to the error message displayed during deployment in BAS #1257

Adds additional note for a customer to replace the host in the URL.

### CODE REVIEW

File: packages/axios-extension/src/abap/message.ts [hunk 1]
```diff
@@ -91,13 +102,29 @@ export function prettyPrintMessage({ msg, log, host }: { msg: string; log: Logge
...
+    isDest?: boolean;
+}): void {
     if (host && path) {
         const base = new URL(host).origin; // We only care for the origin value
         // Add this instruction to the user because of this bug in VS Code: https://github.com/microsoft/vscode/issues/144898
         // It undoes the encoding of special characters in the URL sent to the browser
         log.info('Please copy/paste this URL in a browser for more details:');
+        if (isDest) {
+            log.info(
+                '(Note: As the destination uses a virtual host, you will need to replace the host in the URL with the internal host that is configured with your SAP Cloud Connector)'
```
Comment (longieirl) [inline packages/axios-extension/src/abap/message.ts:125]: @theStiggySpd this info message needs to be changed. Yes, we know its a destination but the URL defined in the destination properties, is not always a virtual host. Its only a virtual host when its on-premise. A suggestion might be;
```
(Note: You will need to replace the host in the URL with the internal host, if your destination is configured using an On-Premise SAP Cloud Connector)
```
In alot of cases, the URL defined is a direct mapping to the internal host mapping defined in the SCC but there is not always valid.

File: packages/axios-extension/test/abap/message.test.ts [hunk 2]
```diff
@@ -20,9 +21,9 @@ describe('message helpers', () => {
 
             const infoMock = (log.info = jest.fn());
             const warningMock = (log.warn = jest.fn());
-            prettyPrintMessage({ msg: JSON.stringify(msg), log, host });
-            // log main message, two messages for the full url, and each detail
-            expect(infoMock).toBeCalledTimes(4);
+            prettyPrintMessage({ msg: JSON.stringify(msg), log, host, isDest });
```
Comment (tobiasqueck) [inline packages/axios-extension/test/abap/message.test.ts:24]: I don't this test is valuable - not your change, the test itself already. I don't really think we need to check how often `log.info` is called.
Could we just change this one to check if infoMock is called (no matter how often) and have an additional test where we use a url matching `.dest` and check if the note has been logged?

**Review by tobiasqueck — COMMENTED** [review]
* code looks good
* changeset is correct
* small comment about the test that could be improved - but I am ok if you reject that comment ;)

File: packages/axios-extension/src/abap/message.ts (hunk 1 above)
Comment (mikicvi-SAP) [reply packages/axios-extension/src/abap/message.ts]: Adjusted the message, if @theStiggySpd approves

File: packages/axios-extension/test/abap/message.test.ts (hunk 2 above)
Comment (mikicvi-SAP) [reply packages/axios-extension/test/abap/message.test.ts:23]: Thank you for the review.
Please review the new changes.

**Review by tobiasqueck — APPROVED** [review]
* changeset ok
* code is clean and easy to understand
* requested changes to the test have been applied

File: packages/axios-extension/src/abap/message.ts (hunk 1 above)
Comment (theStiggySpd) [reply packages/axios-extension/src/abap/message.ts]: Looks good @longieirl @mikicvi-SAP

**Review by devinea — APPROVED** [review]
changeset, msg text, code and tests are good.

**Pull request merged**